integration/tests/conftest.py

```diff
@@ -235,3 +235,11 @@ def meta_proxy_contract(web3_client, faucet, class_account):
         "./EIPs/MetaProxy.sol", "0.8.10", account=class_account
     )
     return contract
+
+
+@pytest.fixture(scope="class")
+def event_caller_contract(web3_client, class_account) -> typing.Any:
```

Comment:
Are you sure that ./tests/services/test_ws_subscriver.py have access to that conftest.py?

Reply:
Yes, it's working, as conftest located in the parent directory